include/MeshKernel/Mesh.hpp

```diff
@@ -91,6 +91,13 @@ namespace meshkernel
             AdministrateMeshEdgesAndFaces
         };
 
+        /// Enumerator describing the different mesh types
+        enum class MeshTypes
```

Comment:
isn't this enum only used in testing code, such as MakeMeshes.* ? if so, you could define it there. Unless you foresee a future need for it...